app/helpers/application_controller_helper.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
```

Comment:
You can actually delete this file now that you renamed it to most_focus_helper.rb